Tests/Contour.RabbitMq.Tests/BasicRequestResponseSpecs.cs

```diff
@@ -557,6 +557,69 @@ public void should_receive_successfully()
             }
         }
 
+        [TestFixture]
+        [Category("Integration")]
+        public class WhileProcessingIncomingRequest : RabbitMqFixture
+        {
+            /// <summary>
+            /// Ответы должны приходить.
+            /// </summary>
+            [Test]
+            public void ShouldSuccessfullyMakeConcurrentRequestsToSingleRemoteConsumer()
+            {
+                IBus producer = this.StartBus(
+                    "producer",
+                    cfg =>
+                    {
+                        cfg.Route("originalRequest")
+                            .WithDefaultCallbackEndpoint();
+                    });
+
+                var requestingConsumer = this.StartBus(
```

Comment:
Because without initial request there's no correlation id saved in CallContext and all requests will have different correlation ids and issue https://github.com/SDVentures/Contour/issues/25 won't be reproduced.